### PR TITLE
Add Interspeech paper as attachment

### PR DESCRIPTION
**Pull request by hagenw**
This adds the Interspeech paper from 2005 as attachment to the database.

### CODE REVIEW

File: 1.4.0/update.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    name,
+    version=previous_version,
+    num_workers=8,
+    only_metadata=True,
+    cache_root=None,
+    verbose=True,
+)
+
+# Download paper
+rel_paper_path = 'extra/burkhardt2005.pdf'
```
[the remark this answers is not in the document]
Comment (hagenw) [reply 1.4.0/update.py]: Changed to `docs/`

File: 1.4.0/update.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+# Download paper
+rel_paper_path = 'docs/burkhardt2005.pdf'
+abs_paper_path = audeer.path(build_dir, rel_paper_path)
+audeer.mkdir(os.path.dirname(abs_paper_path))
+audeer.download_url(db.meta['pdf'], abs_paper_path)
+
+# Add paper as attachment
+db.attachments['paper'] = audformat.Attachment(
+    rel_paper_path,
+    description='Interspeech paper about emodb.',
```
Comment (frankenjoe) [inline 1.4.0/update.py:32]:
```suggestion
    description='INTERSPEECH conference paper on emodb.',
```

**Comment by frankenjoe** [comment]
Would it make sense to also add the bibtex file?

**Comment by hagenw** [comment]
Or maybe only the bibtex file?

**Comment by frankenjoe** [comment]
Yes, maybe that makes most sense, as we already provide a link to the pdf.

**Comment by hagenw** [comment]
I replaced the paper now by a BIB file containing the BibTeX entry for the Interspeech paper.

**Comment by frankenjoe** [comment]
Ups, I was too fast, the PR title was still saying "paper" :)

**Comment by hagenw** [comment]
I will fix this and update the commit in the `main` branch.

**Pull request merged**